src/ConfigR/Scripting/ScriptConfigLoader.cs

```diff
@@ -64,25 +64,25 @@ private static void RethrowExceptionIfAny(ScriptResult result, string scriptPath
         {
             if (result.CompileExceptionInfo != null)
             {
-                result.CompileExceptionInfo.Throw();
-            }
-
-            if (result.ExecuteExceptionInfo != null)
-            {
                 // HACK: waiting on https://github.com/scriptcs/scriptcs/issues/545

```

Comment:
I believe it is.

The problem now is that `CSharpScriptInMemoryEngine` still throws a compilation exception in the case of a no-op file, which means that, in 0.16.1, `scriptcs empty.csx` works, but `scriptcs empty.csx -debug` doesn't.